client/configuration/webpack/webpack.config.dev.js

```diff
@@ -73,6 +80,44 @@ module.exports = {
       inject: true,
       template: path.resolve("index.html"),
     }),
+    new ObsoleteWebpackPlugin({
+      name: "obsolete",
```

Comment:
tiny nit:

I see that the template is used in both `config.dev.js` and `config.prod.js`

In the spirit of DRY and making the template easier to maintain/update the template down the road, can we try to extract the template as an html file and use Webpack's [HTML Loader](https://webpack.js.org/loaders/html-loader/) to load the template as string and include it here? (I've never done this before, but looks possible lol)

E.g.,

1. Move the template into a file `unsupported-browser.html`
2. Import the file in this config file

```ts
const unsupportedBrowserTemplate = require('unsupported-browser.html')

new ObsoleteWebpackPlugin({
  ...
  template: unsupportedBrowserTemplate,
  ...
})
```
3. We also need to add the html-loader plugin to our config files too:

```ts
module.exports = {
  module: {
    rules: [
      {
        test: /\.html$/i,
        loader: 'html-loader',
      },
    ],
  },
};
```
---

Lastly, this is probably a separate ticket, but I'm thinking given that dev and prod config share a few common settings, maybe we can extract the common ones out, and import the common part into dev and prod config files and use [webpack-merge](https://github.com/survivejs/webpack-merge)?

E.g., Meta example:

1. Common: https://github.com/chanzuckerberg/khepri/blob/develop/webpack/webpack.common-client.js
2. Dev: https://github.com/chanzuckerberg/khepri/blob/develop/webpack/webpack.client-dev.js
3. Prod: https://github.com/chanzuckerberg/khepri/blob/develop/webpack/webpack.client-prod.js